DJHelper/DJHelperUITests/DJHelperUITests.swift

```diff
@@ -10,25 +10,18 @@ import XCTest
 
 class DJHelperUITests: XCTestCase {
 
-    // Login with invalid credentials should present alert
-    // Login with valid credentials should transition to trable view controller
-    // Tapping Register on Sign In scene should transition to Register scene
-    // Register with different password fields should present password alert
+
     // Register with empty text field should return nothing
-    // Register with a unique username and matching passwords should present success alert
-    // ...and continuing to sign in should transition to table view controller.
 
     let app = XCUIApplication()
     override func setUp() {
         continueAfterFailure = false
         app.launch()
     }
 
+    // Login with invalid credentials should present alert
     func testInvalidLogIn() {
 
-        // TODO: This test helped me to find an error!
-        // We need to add an alert if the fetch request does not find the username in core data.
```

Comment:
Yes we should talk about how we are going to handle errors for a consistent theme throughout the app.